src/CMakeLists.txt

```diff
@@ -31,7 +31,11 @@ pkg_check_modules (LIBGVM_BASE REQUIRED libgvm_base>=22.17)
 pkg_check_modules (LIBGVM_UTIL REQUIRED libgvm_util>=22.17)
 pkg_check_modules (LIBGVM_OSP REQUIRED libgvm_osp>=22.17)
 pkg_check_modules (LIBGVM_GMP REQUIRED libgvm_gmp>=22.17)
-pkg_check_modules (LIBGVM_OPENVASD REQUIRED libgvm_openvasd>=22.17)
+if (OPENVASD)
+    pkg_check_modules (LIBGVM_OPENVASD REQUIRED libgvm_openvasd>=22.17)
+else (OPENVASD)
+    message (STATUS "OPENVASD flag is not enabled")
```

Comment:
Shouldn't be this a failure? if somebody enables OPENVASD he wants to build with it. and if that isn't possible because some dependency isn't installed the configure step should fail in my opinion.

Reply:
It think it does this already: If you enable the OPENVASD flag the configuration will fail if you don't have the library installed because the pkg_check_modules has the REQUIRED option.